src/woo_publications/publications/tests/test_index_publication_task.py

```diff
@@ -0,0 +1,62 @@
+from django.test import TestCase
+
+from woo_publications.config.models import GlobalConfiguration
+from woo_publications.contrib.tests.factories import ServiceFactory
+from woo_publications.utils.tests.vcr import VCRMixin
+
+from ..constants import PublicationStatusOptions
+from ..tasks import index_publication
+from .factories import PublicationFactory
+
+
+class IndexPublicationTaskTests(VCRMixin, TestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        service = ServiceFactory.create(for_gpp_search_docker_compose=True)
+        GlobalConfiguration.objects.update_or_create(
+            pk=GlobalConfiguration.singleton_instance_id,
+            defaults={"gpp_search_service": service},
+        )
+
+    def setUp(self):
+        super().setUp()
+
+        GlobalConfiguration.clear_cache
```

Comment:
```suggestion
```

Or `GlobalConfiguration.clear_cache()` if you want to make sure this is run before each test as well